Helper/Order.php

```diff
@@ -801,6 +801,13 @@ public function setOrderPaymentInfoData($payment, $transaction)
         if (!empty($transaction->from_credit_card->token_type) && $transaction->from_credit_card->token_type == "applepay") {
             $payment->setAdditionalData($transaction->from_credit_card->token_type);
         }
+
+        if (!empty($transaction->processor) && $transaction->processor == "stripe" && !empty($transaction->authorization_id)){
```

Comment:
Looks great, but other platforms can have their own authorization_id.
What if we call it something like payment_processor_transaction_id and save for all processors?
CC: @pranamya-nayak

Reply:
@vitaliyreznikov  I think that's a good idea. I've updated the code.